types/office-js-preview/index.d.ts

```diff
@@ -10564,6 +10564,56 @@ declare namespace Office {
          * @beta
          */
         getInitializationContextAsync(callback?: (asyncResult: Office.AsyncResult<string>) => void): void;
+        /**
+         * Asynchronously gets the ID of a saved item.
+         *
+         * When invoked, this method returns the item ID via the callback method.
+         * 
+         * **Note**: If your add-in calls `getItemIdAsync` on an item in compose mode in order to get an `itemId` to use with EWS or the REST API,
+         * be aware that when Outlook is in cached mode, it may take some time before the item is actually synced to the server.
+         * Until the item is synced, using the `itemId` will return an error.
```

Comment:
Consider "Until the item is synced, `itemId` returns an error." It's less wordy.

Also, consider adding the error name here.